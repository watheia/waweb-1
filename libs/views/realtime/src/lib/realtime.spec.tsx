import { render } from '@testing-library/react';

import Realtime from './realtime';

describe('waweb.views/realtime', () => {
  it('should render successfully', () => {
    const { container } = render(<Realtime />);
    expect(container).toBeInstanceOf(HTMLElement);
    expect(container).toMatchSnapshot();
  });
});
