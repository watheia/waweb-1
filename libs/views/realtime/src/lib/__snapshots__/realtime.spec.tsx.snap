// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`waweb.views/realtime should render successfully 1`] = `
<div>
  <div
    class="root"
  >
    <h1
      class="appear appear-third hero"
    >
      Watheia
      <br
        class="show-on-desktop"
      />
       Realtime
    </h1>
  </div>
</div>
`;
